### ISSUE TITLE
Commands not added to $PATH

### ISSUE THREAD

**Issue by simeg** [rcaloras/bashhub-client#53]
I just installed bashhub on Arch Linux and the installation went fine but the commands were not added to $PATH. To resolve this I ran

```sh
ln -s $HOME/.bashhub/env/bin/bh /usr/bin/
```

```sh
ln -s $HOME/.bashhub/env/bin/bashhub /usr/bin/
```

Now everything works as it should! Just thought I'd leave this here in case someone else has the same problem.

**Comment by rcaloras** [comment]
@simeg Hmm that's a little strange. How did you run install? It was via `install-bashhub.sh`? Was it for bash or zsh? 

Typical install should create a new bin directory and then link it for you during sourcing.
Linking during install:
https://github.com/rcaloras/bashhub-client/blob/master/install-bashhub.sh#L259-L260

Adding to `$PATH`:
https://github.com/rcaloras/bashhub-client/blob/master/bashhub/shell/bashhub.sh#L8

**Comment by simeg** [comment]
I noticed today that bashhub was not capturing my commands, and it was saying it could not find the function adding them to `$PATH` (`__bh_path_add()`) when I sourced my `.zshrc`. I did some debugging and I noticed I had used the `bash` installation script instead of the one for `zsh`. I uninstalled it and then installed it properly this time and then it worked.

Since my symlinks were already setup I didn't see a difference, but I guess this would've resolved the problem.

(This is why I added the zsh installation part on the wiki: https://github.com/rcaloras/bashhub-client/wiki/Install)
